RecoJets/JetProducers/python/ak4PFJets_cfi.py

```diff
@@ -11,3 +11,25 @@
     rParam       = cms.double(0.4)
```

Comment:
How about turning on "doAreaFastjet=True" for ak4PFJets (and thus all other derived ak jet algorithms) during this cleanup. This is currently done only in the PAT/MiniAOD configuration and lead to many errors in the past.